pkg/api/grpcServer_test.go

```diff
@@ -459,7 +466,7 @@ func TestRudeClose(t *testing.T) {
 }
 
 func TestBrokenStream(t *testing.T) {
-	ts, err := prepTestState(30000)
+	ts, err := prepTestState(30009)
```

Comment:
This is not bazel, this is the go test infrastructure. Multiple individual tests in a single test program can be run in parallel and so they _do_ get port conflicts (that's why the code was the way it was).

I didn't know that a 0 port would give a valid port automatically. So I've updated the code accordingly.